docs/code/uvwget/defs.h

```diff
@@ -0,0 +1,10 @@
+#ifndef DEFS_H_
+#define DEFS_H_
+
+#include <stddef.h> 
+
+#define CONTAINER_OF(ptr, type, field)                                        \
+  ((type *) ((char *) (ptr) - offsetof(type, field)))
```

Comment:
uv-common.h defines this as:
```c
#define container_of(ptr, type, member) \
  ((type *) ((char *) (ptr) - offsetof(type, member)))
```

should we make this lowercase? otherwise, lgtm.